clients/privacy-center/cypress/e2e/consent-i18n.cy.ts

```diff
@@ -1,125 +1,361 @@
-    accept_button_label: "Participar en todo",
-    acknowledge_button_label: "Aceptar",
-    gpc_title: "Control de privacidad global detectado",
-    gpc_description:
-      "Su preferencia de control de privacidad global se ha respetado.",
-    privacy_policy_link_label: "Política de privacidad",
-    privacy_policy_url: "https://privacy.example.com/",
-  };
+/**********************************************************
+ *
+ * EXPECTED TRANSLATION STRINGS
+ *
+ * NOTE: There is a *lot* of duplication here between test data, fixture data,
+ * and the static translations in fides.js project. We *could* automatically
+ * import those strings here... but it's much easier to *read* these tests when
+ * the expected strings are all here. This also helps us catch any accidental
+ * changes to strings or misspellings!
+ *
+ **********************************************************/
+
+/**
+ * English translations for banner & modal
+ */
+const ENGLISH_BANNER: TestBannerTranslations = {
+  banner_title: "[banner] Manage your consent preferences",
+  banner_description: "[banner] We use cookies and similar",
+  privacy_preferences_link_label: "Manage preferences",
+  reject_button_label: "Opt out of all",
+  accept_button_label: "Opt in to all",
+  acknowledge_button_label: "OK",
+  gpc_label: "Global Privacy Control",
+  gpc_status_label: "Applied",
+  privacy_policy_link_label: "Privacy Policy",
+  privacy_policy_url: "https://privacy.example.com/",
+};
+
+const ENGLISH_MODAL: TestModalTranslations = {
+  title: "Manage your consent preferences",
+  description: "We use cookies and similar",
+  save_button_label: "Save",
+  reject_button_label: "Opt out of all",
+  accept_button_label: "Opt in to all",
+  acknowledge_button_label: "OK",
+  gpc_title: "Global Privacy Control detected",
+  gpc_description: "Your global privacy control preference has been honored.",
+  gpc_label: "Global Privacy Control",
+  gpc_applied_label: "Applied",
+  gpc_overridden_label: "Overridden",
+  privacy_policy_link_label: "Privacy Policy",
+  privacy_policy_url: "https://privacy.example.com/",
+};
 
-  const SPANISH_NOTICES: TestNoticeTranslations[] = [
-    {
-      title: "Mercadotecnia",
-      description: "Este sitio web usa cookies y servicios de mercadotecnia",
-    },
-    {
-      title: "Análisis",
-      description: "Este sitio web usa cookies analíticas y servicios",
-    },
-    {
-      title: "Esenciales",
-      description: "Este sitio web utiliza cookies esenciales y servicios",
-    },
-  ];
-
-  // Setup a test case with the given params
+const ENGLISH_NOTICES: TestNoticeTranslations[] = [
+  { title: "Advertising", description: "This website uses marketing" },
+  { title: "Analytics", description: "This website uses analytics" },
+  { title: "Essential", description: "This website uses essential" },
+];
+
+/**
+ * English translations for TCF banner & modal
+ */
+const ENGLISH_TCF_BANNER: TestTcfBannerTranslations = {
```

Comment:
OK, these are new in this PR: the `ENGLISH_TCF_*` and `SPANISH_TCF_*` expected results for translated strings in the UI.